inc/class-upgrade.php

```diff
@@ -556,6 +560,18 @@ private function upgrade_74() {
 		$this->remove_sitemap_validators();
 	}
 
+	/**
+	 * Performs the 7.6 upgrade.
+	 *
+	 * @return void
+	 */
+	private function upgrade_76() {
+		global $wpdb;
+
+		// Remove all OpenGraph content image cache.
+		$wpdb->query( "DELETE FROM $wpdb->postmeta WHERE meta_key = '_yoast_wpseo_post_image_cache'" );
```

Comment:
I was seeing this and thought maybe WordPress has something for this, builtin. And yes WordPress does. Maybe we should consider using `delete_post_meta_by_key`. This function also clears the cache. 

If we do this, we should other equivalent logic elsewhere in this class.